README.md

```diff
@@ -136,3 +136,40 @@ If you do not have Python 3.10 installed, you can use root-less podman with a Py
 ```bash
 podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt docker.io/library/python:3.10 bash -c 'pip install nox ; nox -s pip-compile'
 ```
+
+## Creating release tags
+
+When a tag is created in the [`ansible/ansible`](https://github.com/ansible/ansible) repository for a release or release candidate, a corresponding tag should be created in this `ansible-documentation` repository.
+
+First, install the additional tagging dependencies from this repository as follows, creating or activating a `venv` as needed:
+
+```bash
+python3 -m venv ./venv
+source ./venv/bin/activate
+pip install -r hacking/tagger/requirements.txt
+```
+
+Next, ensure that you have both [`ansible/ansible`](https://github.com/ansible/ansible) and [`ansible/ansible-documentation`](https://github.com/ansible/ansible-documentation) repositories checked out. The tool assumes that both checkouts have the same parent directory. You can set different paths to your checkouts with the `--docs` and `--core` options if you have them setup another way.
+
+Lastly, run the appropriate command(s) with the tagger script.
+
+From  your `ansible-documentation` directory, search for tags in the `ansible-core` repository that are missing from the `ansible-documentation` repository.
+
+```bash
+./hacking/tagger/tag.py new-tags
+```
```

Comment:
Running the `tag` subcommand directly should do the job. It just exits normally if there is nothing to tag. `new-tags` is mainly there for debugging and informational purposes, so I'm not sure it makes sense to demonstrate it here. The goal of the script is to provide a *single* idempotent command that does everything for all the actively supported branches without the user having to perform extra steps or decision making.

Reply:
Thanks for the insight on this. I'll tune this up to align with that.